toolium/test/utils/test_dataset_replace_param.py

```diff
@@ -438,3 +443,26 @@ def test_replace_param_partial_string_with_length():
     assert param == 'aaaaa is string'
     param = replace_param('parameter [STRING_WITH_LENGTH_5] is string')
     assert param == 'parameter aaaaa is string'
+
+
+def test_replace_param_replace():
+    param = replace_param('[REPLACE:https://url.com::https::http]')
+    assert param == "http://url.com"
+    param = replace_param('[REPLACE:https://url.com::https://]')
+    assert param == "url.com"
+
+
+def test_replace_param_title():
+    param = replace_param('[TITLE:hola hola]')
+    assert param == "Hola Hola"
+    param = replace_param('[TITLE:holahola]')
+    assert param == "Holahola"
+    param = replace_param('[TITLE:hOlA]')
+    assert param == "HOlA"
+
+
+def test_replace_param_round():
+    param = replace_param('[ROUND:7.5::2]')
+    assert param == 7.5
+    param = replace_param('[ROUND:3.33333333::3]')
+    assert param == 3.333
```

Comment:
Estos los movería debajo de los tests del TODAY y el NOW, al estar en el mismo grupo de transformaciones.

Añadiría también algún caso con infer_param_type=False, para comprobar el efecto:

```suggestion
def test_replace_param_round_with_type_inference():
    param = replace_param('[ROUND:7.5::2]')
    assert param == 7.5
    param = replace_param('[ROUND:3.33333333::3]')
    assert param == 3.333


def test_replace_param_round_without_type_inference():
    param = replace_param('[ROUND:7.500::2]', infer_param_type=False)
    assert param == '7.50'
    param = replace_param('[ROUND:3.33333333::3]', infer_param_type=False)
    assert param == '3.333'
```